protected/humhub/events/MigrationEvent.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * @link      https://www.humhub.org/
+ * @copyright Copyright (c) 2023 HumHub GmbH & Co. KG
+ * @license   https://www.humhub.com/licences
+ */
+
+namespace humhub\events;
+
+use humhub\components\Event;
+use humhub\interfaces\ApplicationInterface;
+use yii\base\Module;
+
+/**
+ * @property int|null $result Result of the migration:
```

Comment:
Can be removed?

I would prefer a status code instead the fixed `errorMessage`.

Reply:
What do you mean by a fixed `errorMessage`?

Firstly, the $result variable is already part of the `humhub\components\Event`. So I can't remove it.
Secondly, the `$result` _is_ filled with a status code:
- `ExitCode::OK`: which is integer `0` and stands for Success
- `ExitCode::UNSPECIFIED_ERROR`: which is integer `1` which stands for failure;
- `Null`: nothing done (e.g. no migrations directory or no migrations pending)

Reply:
Aah ok, missed that.

So the default `result` attribute in the Event class, indicates the Migration success status?

Reply:
That's correct.

Currently, the MigrateController always returns `ExitCode::OK` or `ExitCode::UNSPECIFIED_ERROR`. So in theory we could translate them into a bool (`true`: success, `false`: error, `null`: nothing migrated)